tasks/collect_logs.yml

```diff
@@ -0,0 +1,55 @@
+- name: Ensure local logs dir exists
+  connection: local
+  file:
+    state: "{{ item }}"
+    path: "{{ rhv_he_deploy_log_dir }}"
+  # remove then recreate log dir
+  with_items:
+    - absent
+    - directory
+- name: Check if remote logs dir exists
+  stat:
+    path: /var/log/ovirt-hosted-engine-setup
+  register: rhv_he_deploy_remote_log_dir
+- name: "Write log to {{ rhv_he_deploy_log_dir }} on controller"
+  connection: local
+  copy:
+    content: "{{ item.value }}"
+    # e.g. hosted-engine-deploy-stdout.YYYY-MM-DDTHH:MM:SSZ.log
+    dest: "{{ playbook_dir }}/logs/hosted-engine-deploy-{{ item.key}}.\
```

Comment:
Is `{{ playbook_dir }}` supposed to be `{{ rhv_he_deploy_log_dir }}`?

Reply:
Yuuuuuuuuuuuup!